server/basicServices/basicServices/utility/LogicalTerminationPoint.js

```diff
@@ -31,7 +32,57 @@ exports.resolveHttpTcpAndOperationClientUuidOfNewRelease = function () {
     })
 }
 
+exports.resolveHttpClientUuidOfNewRelease = function () {
+    return new Promise(async function (resolve, reject) {
+        let regardApplicationForwardingName = "PromptForBequeathingDataCausesTransferOfListOfApplications";
+        let registerApplicationForwardingName = "PromptForBequeathingDataCausesTransferOfListOfAlreadyRegisteredApplications";
```

Comment:
The first forwarding name is used in 5 applications (ALT, AA, EATL, OL, OKM), the second one is only used in RO. What is missing here is one used in TAR: PromptForBequeathingDataCausesTransferOfListOfAlreadyGrantedTypeApprovals . Given that there are different names for this forwarding (that according to spec does the exact same thing) I would either:
1. change the spec to use the same forwarding name for all applications
2. move this method to individual applications and handle only one forwarding

Reply:
@PrathibaJee  I think the design here needs your decision.

Reply:
@DanaSunal Thank you for the comment . TAR is missing because in case of TAR regard-application works differently and this issue is not observed there.

Reply:
Thanks Dana and Vanitha.
Thanks Vanitha for including a prototype to handle this mechanism , https://github.com/openBackhaul/AdministratorAdministration/pull/256 .
So , the hardcoded forwardings names are removed , and the forwarding names will be provided from the individual applications.